flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/functions/bridging/BridgingSqlAggFunction.java

```diff
@@ -63,6 +64,7 @@ public final class BridgingSqlAggFunction extends SqlAggFunction {
     private final List<RelDataType> paramTypes;
 
     private BridgingSqlAggFunction(
+            FlinkContext context,
```

Comment:
What is the reason for those changes? Is it only to be able to create the `equalityHandler`? If so, let's keep it simple and remove those.

Reply:
Yes, I'll revert that changes.
Thanks.